wazuh/wazuh-dashboard-plugins
# Fix permissions for group edit buttons

### Description
This PR fixes a bug on which different buttons on the `Endpoint groups` section showed as enabled even if the user had no permissions to use them 
 
### Issues Resolved
#7055
 
### Evidence
![image](https://github.com/user-attachments/assets/b923bfb2-3d75-4641-ad6d-596ff89eabe0)
![image](https://github.com/user-attachments/assets/fb57d4e5-ac51-4221-ba8b-2e860034d41f)
![image](https://github.com/user-attachments/assets/dcdc8aee-96d2-47cc-a0fe-480016db38f1)


### Test
- Generate an environment with a real manager and an agent
- Follow the [documentation](https://documentation.wazuh.com/current/user-manual/user-administration/rbac.html#creating-and-setting-a-wazuh-read-only-user).
- Log in with the readonly user and navigate to `Endpoint groups` and verify that the pencil button in the `Actions` column is disabled
- Click on a group and verify that on the `Agents` tab, the `Manage agents` button is disabled
- Navigate to the `Files tab` and verify that the `Edit group configuration` button, as well as any pencil icon on the `Actions` column is disabled.
- With an administrator user, repeat the previous steps and verify that the buttons work as expected 

>[!NOTE]
>It is important to verify that the API is set to `run_as` mode, and to make sure that the RBAC configuration is done with the `manager` API selected. It will not work with imposter.

### Check List
